packages/example/pages/_app.tsx

```diff
@@ -4,6 +4,7 @@ import './global.css';
 import {
   AvatarComponent,
   DisclaimerComponent,
+  Language,
```

Comment:
I think we can remove the need for exporting Language; the type exposed in the RainbowKitProvider prop should be enough, maybe? There shouldn't be extensive casting needs.

Reply:
Good catch. I have update on that. In most of typescript projects when using `enum` types the prop that requires the `enum` type will throw error if you don't import and specify it like this:

![image](https://github.com/rainbow-me/rainbowkit/assets/53529533/c9f775d6-655e-4747-8824-9022543de275)

VS

![image](https://github.com/rainbow-me/rainbowkit/assets/53529533/a0ab41b8-929d-41b7-b290-206234e85425)

Reply:
Some libraries like alchemy-sdk also provides `enum` types in their library to avoid type errors 👻

Reply:
Snooped at other approaches here. I think we can accept the raw string directly like "en-US". How about we expose a union type for that prop similar to `Theme` and export as a type rather than an enum
```ts
export type Locale =
  | 'en-US'
  | 'es-419'
  | 'fr-FR'
  | 'ja-JP'
  | 'pt-BR'
  | 'zh-CN'
  | 'id-ID'
  | 'hi-IN'
  | 'tr-TR'
  | 'ru-RU';
```
Researched a bit more about Next.js, and I think it makes sense for us to compatible with their `en-US` format instead of `en_US` externally. But we should keep using our format internally. I think we'd should have to bring your switch case into the repo which would be good to simplify the example too.